index.html

```diff
@@ -213,40 +102,41 @@ <h5 class="btn-connect">LET’S CONNECT</h5>
             <div class="line"></div>
         </div>
     </div>
-    </section>
-    <!-- footer section start -->
-    <section id="contact-section" class="footer-section">
-        <footer class="footer-form">
-            <div class="footer-bg">
-                <div class="form-heading">
-                    <h2 class="contact-heading">Contact me </h2>
-                    <p class="contact-detail">If you have an application you are
-                        interested in developing, a feature that
-                        you need built or a project that needs
-                        coding. I’d love to help with it
-                    </p>
-                </div>
+</section>
+<!-- footer section start -->
+<section id="contact-section" class="footer-section">
+    <footer class="footer-form">
+        <div class="footer-bg">
+            <div class="form-heading">
+                <h2 class="contact-heading">Contact me </h2>
+                <p class="contact-detail">If you have an application you are
+                    interested in developing, a feature that
+                    you need built or a project that needs
+                    coding. I’d love to help with it
+                </p>
+            </div>
 
-                <form action="https://formspree.io/f/xlevdjbz" method="post">
-                    <ul>
-                        <li>
-                            <input class="name" type="text" maxlength="30" name="name" placeholder="Full Name" required>
-                        </li>
-                        <li>
-                            <input class="email" type="email" name="email" placeholder="Email Address" required>
-                        </li>
-                        <li>
+            <form id="form" action="https://formspree.io/f/xlevdjbz" method="post">
+                <ul>
+                    <li>
+                        <input class="name" type="text" maxlength="30" name="name" placeholder="Full Name" required>
+                    </li>
+                    <li>
+                        <input class="email" id="email" type="email" name="email" placeholder="Email Address" required>
+                    </li>
+                    <li id="error-msg"></li>
```

Comment:
- [ ] Great Job the validations works perfectly , please note that the error message should be close to the submit button

![image](https://user-images.githubusercontent.com/89282221/190854416-7f8508e9-bfcc-43a3-ac18-39a7fa0cb089.png)

Reply:
I applied the above mention issue.